src/helpers/getSortedTodos.ts

```diff
@@ -0,0 +1,39 @@
+/* pretie-disable */
```

Comment:
It should be `/* prettier-ignore */` according to https://prettier.io/docs/en/ignore.html. But it's better not to disable code style checks if there's no strong reason for that.

Reply:
Got it) I did not have the possibility to deploy the project because of this error. So I turned it off before deploy)